### DISCUSSION TITLE
Performance tools (Lighthouse) show very slow "Time to Interactive" on my Gatsby site. How can I improve it?

### DISCUSSION THREAD

**Discussion by lumenwrites** [gatsbyjs/gatsby#26039] [Help] [answered]
Hello! I'm looking for some help. My [website](https://playmirage.io/) is getting a very low time to interactive score (over 10 seconds). 

It tells me I need to "Minimize main-thread work" and "Reduce JavaScript execution time".

Could you share some advice/tips on what I can do? How do you improve these metrics in your projects?

From googling, it seems like the main thing people do is code splitting, but Gatsby does that automatically already, right?

What else can be done?

**Comment by paulie-rodriguez** [comment, marked as answer]
#25722 touches base on this and the [comment](https://github.com/gatsbyjs/gatsby/issues/24332#issuecomment-643504322) linked on that thread additionally touch base that this seems to have to do with lighthouse v6.1 which was released very recently now has diagnostics for main-thread blocking tasks. Even the official gatsbyjs.org site have lower lighthouse scores now since the lighthouse updates it seems.   
Your site, and gatsby sites are still fast, but the score taking such a big hit is definitely daunting.
